Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -64,7 +64,10 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
 
             if (user.get('avatar')) {
                 contentService.loadImageVariation(user.get('avatar').variations.platformui_profileview.href, tasks.add(function (error, response) { //jshint ignore:line
-                    if ( error ) {
+                    if ( error && error.message.indexOf('401') !== -1 ) {
```

Comment:
that's very bad way of checking the HTTP status code... You can check it in `response.status`.